portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase

```diff
@@ -108,9 +108,33 @@ definition {
 	@ignore = "true"
```

Comment:
If the test is running, you have to remove this line so the test can be executed in CI.

Reply:
Yes, i forgot about that. I'll fix it.